spec/models/issue_spec.rb

```diff
@@ -0,0 +1,118 @@
+require 'rails_helper'
+
+RSpec.describe Issue, type: :model do
+  before(:each) do
+    @issue = Issue.new(reporter_id: 321,
```

Comment:
mb it's better to use subject here https://www.relishapp.com/rspec/rspec-core/v/3-5/docs/subject/explicit-subject